example/README.md

```diff
@@ -50,24 +49,24 @@ You can find your API Gateway Endpoint URL in the output values displayed after
 Build your application with the `sam build` command.
 
 ```bash
-sam-app$ sam build
+examples$ sam build
 ```
 
-The SAM CLI installs dependencies defined in `HelloWorldFunction/pom.xml`, creates a deployment package, and saves it in the `.aws-sam/build` folder.
+The SAM CLI installs dependencies defined in `HelloWorldFunction/build.gradle`, creates a deployment package, and saves it in the `.aws-sam/build` folder.
```

Comment:
I think I'd rather have both examples than just gradle.

Reply:
Initially I had two separate projects, but because only the build configuration was the difference it makes sense to pull them together in one project and have two options `build.gradle` and `pom.xml`. I will add the documentation to reflect that the user have a choice.

Reply:
Interesting, `sam build` favors gradle over maven, if both configurations are in the same root folder 😄

Reply:
haha .. good to know that :D